.github/workflows/build_test.yml

```diff
@@ -76,7 +77,7 @@ jobs:
         run: |
           sudo systemctl start mysql
           mysql -u${{env.KBS_DB_USER}} -p${{env.KBS_DB_PW}} -e 'CREATE DATABASE ${{env.KBS_DB}};'
-          mysql -u${{env.KBS_DB_USER}} -p${{env.KBS_DB_PW}} ${{env.KBS_DB}} < db.sql
+          mysql -u${{env.KBS_DB_USER}} -p${{env.KBS_DB_PW}} ${{env.KBS_DB}} < db-mysql.sql
```

Comment:
So we are only testing for mysql? That seems reasonable but I am a little bit worried about people (like future me) forgetting to update the other db files.

Reply:
I will update this so that we test all 3 backends.

Reply:
Github workflows now test all 3 database backends.  Our current code is passing tests on all 3 backends.